<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileConversionPanel</class>
 <widget class="QWidget" name="FileConversionPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>本地文件转换</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="SelectFileBtn">
           <property name="text">
            <string>选择文件</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="SelectAllBtn">
           <property name="text">
            <string>全选</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="RefreshBtn">
           <property name="text">
            <string>提交选择</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="deleteBtn">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../src.qrc">
             <normaloff>:/uidemo08/dataCollectingIcon_files/deleteIcon.webp</normaloff>:/uidemo08/dataCollectingIcon_files/deleteIcon.webp</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="FileSelectingTable"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>请选择产品型号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="pipeTypecomboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>请选择产品类型</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>变形</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>漏磁</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="pipeSizecomboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>请选择产品尺寸</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8inch</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10inch</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12inch</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14inch</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="ConvertBtn">
        <property name="text">
         <string>  开始转换</string>
        </property>
        <property name="icon">
         <iconset resource="../../src.qrc">
          <normaloff>:/uidemo08/dataCollectingIcon_files/1-2004121ZI0.png</normaloff>:/uidemo08/dataCollectingIcon_files/1-2004121ZI0.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="SavePathBtn">
        <property name="text">
         <string>选择存储路径</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="SavePathEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../src.qrc"/>
 </resources>
 <connections/>
</ui>
